src/wasm/wasm-emscripten.cpp

```diff
@@ -1275,12 +1287,15 @@ void EmscriptenGlueGenerator::exportWasiStart() {
   // If main exists, export a function to call it per the wasi standard.
   Name main = "main";
   if (!wasm.getFunctionOrNull(main)) {
+    BYN_TRACE("exportWasiStart: main not found\n");
```

Comment:
Not related to this change per se:

Do we have any BYN_TRACEs that don't end with "\n" (or std::endl)? Should we make BYN_TRACE always append a newline?

Reply:
I think I'll leave that for a separate PR and/or discussion.